packages/tocco-ui/src/Pagination/Pagination.spec.js

```diff
@@ -24,47 +24,47 @@ describe('tocco-ui', function() {
+    //   setTimeout(() => {
+    //     expect(onPageChange).to.be.calledWith(2)
+    //     done()
+    //   }, 200)
+    // })
 
-    it('should disable forward and back button depending on current page', () => {
-      const wrapper = shallow(<Pagination totalRecords={30} recordsPerPage={10}/>)
+    // it('should disable forward and back button depending on current page', () => {
+    //   const wrapper = shallow(<Pagination totalRecords={30} recordsPerPage={10}/>)
 
-      expect(wrapper.find('#forwardButton')).to.not.be.disabled()
-      expect(wrapper.find('#toLastButton')).to.not.be.disabled()
-      expect(wrapper.find('#toFirstButton')).to.be.disabled()
-      expect(wrapper.find('#backButton')).to.be.disabled()
+    //   expect(wrapper.find('#forwardButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#toLastButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#toFirstButton')).to.be.disabled()
+    //   expect(wrapper.find('#backButton')).to.be.disabled()
 
-      wrapper.find('#toLastButton').simulate('click')
+    //   wrapper.find('#toLastButton').simulate('click')
 
-      expect(wrapper.find('#forwardButton')).to.be.disabled()
-      expect(wrapper.find('#toLastButton')).to.be.disabled()
-      expect(wrapper.find('#toFirstButton')).to.not.be.disabled()
-      expect(wrapper.find('#backButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#forwardButton')).to.be.disabled()
+    //   expect(wrapper.find('#toLastButton')).to.be.disabled()
+    //   expect(wrapper.find('#toFirstButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#backButton')).to.not.be.disabled()
 
-      wrapper.find('#backButton').simulate('click')
+    //   wrapper.find('#backButton').simulate('click')
 
-      expect(wrapper.find('#forwardButton')).to.not.be.disabled()
-      expect(wrapper.find('#toLastButton')).to.not.be.disabled()
-      expect(wrapper.find('#toFirstButton')).to.not.be.disabled()
-      expect(wrapper.find('#backButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#forwardButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#toLastButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#toFirstButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#backButton')).to.not.be.disabled()
 
-      wrapper.find('#toFirstButton').simulate('click')
+    //   wrapper.find('#toFirstButton').simulate('click')
 
-      expect(wrapper.find('#forwardButton')).to.not.be.disabled()
-      expect(wrapper.find('#toLastButton')).to.not.be.disabled()
-      expect(wrapper.find('#toFirstButton')).to.be.disabled()
-      expect(wrapper.find('#backButton')).to.be.disabled()
-    })
+    //   expect(wrapper.find('#forwardButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#toLastButton')).to.not.be.disabled()
+    //   expect(wrapper.find('#toFirstButton')).to.be.disabled()
+    //   expect(wrapper.find('#backButton')).to.be.disabled()
+    // })
```

Comment:
(we'll fix those tests before we merge the pull request)

Reply:
see aa43b37679098b01ae7c2f0aa9a426190fa0779e